src/Storage/Storage/File/Cmdlet/CloseAzureStorageFileHandle.cs

```diff
@@ -170,6 +170,7 @@ public override void ExecuteCmdlet()
                 switch (this.ParameterSetName)
                 {
                     case DirectoryCloseAllParameterSetName:
+                        CheckContextForObjectInput((AzureStorageContext)this.Context);
```

Comment:
For the close/get file handle cmdlet, we might don't need check context, since the output object has not context.

Reply:
Removed.